src/edge/blastdb.py

```diff
@@ -27,11 +27,11 @@ def fragment_fasta_fn(fragment):
                                                            fragment.id)
 
 
-def build_fragment_fasta(fragment):
+def build_fragment_fasta(fragment, refresh=False):
     fn = fragment_fasta_fn(fragment)
     make_required_dirs(fn)
 
-    if not os.path.isfile(fn):  # have not built this fasta
+    if not os.path.isfile(fn) or refresh is True:  # have not built this fasta
```

Comment:
no need to say `is True`; fine to just evaluate it as boolean: `if … or refresh:`